src/components/user/EditForm.js

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import { func, string, bool, object } from 'prop-types';
+import { Field, reduxForm } from 'redux-form/immutable';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+import {
+  injectIntl,
+  intlShape,
+  defineMessages,
+  FormattedMessage
+} from 'react-intl';
+
+import Routes from '../../constants/routesPaths';
+import Loading from '../common/Loading';
+import Input from '../common/Input';
+import SelectBox from '../common/Select';
+import { validations, edit } from '../../utils/constraints';
+import smilies from '../../img/smilies.svg';
+import close from '../../img/close.png';
+import profilePic from '../../img/guy.svg';
+import blueCircle from '../../img/blue-circle.svg';
+
+const messages = defineMessages({
+  name: { id: 'signup.name' },
+  firstName: { id: 'edit.firstName' },
+  lastName: { id: 'edit.lastName' },
+  email: { id: 'login.form.email' },
+  gender: { id: 'signup.gender' },
+  genderSelect: { id: 'signup.genderSelect' },
+  success: { id: 'edit.success' }
+});
+
+const options = [
+  { value: 'male', label: 'Male' },
+  { value: 'female', label: 'Female' },
+  { value: 'other', label: 'Other' }
+];
+
+let EditForm = ({ initialValues: user, handleSubmit, error, submitting, intl }) => (
+  <div className="left-panel">
+    <div className="top-left">
+      <Link to={Routes.index}>
+        <img src={close} alt="back" className="icon" />
+      </Link>
+    </div>
+    <p className="target-title"><FormattedMessage id="edit.title" /></p>
+    <div className="top-pic" >
+      <img src={blueCircle} alt="blue" />
+      <img src={profilePic} alt="guy" />
+    </div>
+    {user.firstName}
+    <form className="form" onSubmit={handleSubmit}>
+      {error && <strong>{error}</strong>}
+      <div>
+        <Field
+          name="email"
+          label={intl.formatMessage(messages.email)}
+          component={Input}
+          type="email"
+          className="input"
```

Comment:
Maybe you can add this `input` class in the `Input` component root tag instead of adding it every time here?